src/App.tsx

```diff
@@ -1,61 +1,46 @@
+import { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
```

Comment:
The `getUser` function returns `null` if no user is found. Ensure that the `Todo` type can handle a `null` value for the `user` property, or handle this case appropriately in the code to avoid potential runtime errors.